app/models/report_time_point.rb

```diff
@@ -9,19 +9,63 @@ class ReportTimePoint
   include Mongoid::Document
   include Mongoid::Timestamps
 
+  ## Names of defined reports
+  STUDY_COUNTS = {
+    name: 'study_counts',
+    proc: Proc.new { SummaryStatsUtils.study_counts }
+  }
+  WEEKLY_RETURNING_USERS = {
+    name: 'Weekly Returning Users',
```

Comment:
It's internal, but why not match wider convention?

```suggestion
    name: 'Weekly returning users',
```

Reply:
it would require a migration script, since these names are in the database and that report has been running for a while